client/src/components/Form.jsx

```diff
@@ -31,8 +46,8 @@ class Form extends React.Component {
     return <div>
       <h3 id="company">{this.state.company}</h3>
       <MarketPrice prices={this.state.marketPrice}/>
-      <input placeholder="Shares"></input>
-      <EstimatedCost estimation={this.state.estimatedCost}/>
+      <input type="number" placeholder="Shares" onChange={this.handleShares}></input>
```

Comment:
input can be self closing tags for cleaner look. 
Example:
<input type="number" placeholder="Shares" onChange={this.handleShares} />

Reply:
// <input type="number" placeholder="Shares" onChange={this.handleShares}>